docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3.4'
+
+configs:
+  prometheus_config:
+    file: ./TodoApi/prometheus.yml
+
+services:
+  todoapi:
+    image: todoapi
+    pull_policy: if_not_present
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
+      - ASPNETCORE_URLS=https://+:443;http://+:80
+      - OTEL_EXPORTER_OTLP_ENDPOINT=http://jaeger:4317
+    ports:
+      - "5000:80"
+      - "5001:443"
+    volumes:
+      - ${APPDATA}/Microsoft/UserSecrets:/root/.microsoft/usersecrets:ro
+      - ${APPDATA}/ASP.NET/Https:/root/.aspnet/https:ro
```

Comment:
This is windows specific.

Reply:
What happens when this runs on linux? I guess the secrets dont get mapped

Reply:
Yeah. It doesn't get mapped. I will update and push it again with a more generic approach.

Reply:
I end up creating two volumes map. One for Windows and the other for macOS and Linux.